integration-tests/infinispan/src/test/java/org/apache/camel/quarkus/component/infinispan/InfinispanServerTestResource.java

```diff
@@ -16,20 +16,21 @@
  */
 package org.apache.camel.quarkus.component.infinispan;
 
-import java.util.Map;
+import io.quarkus.test.common.QuarkusTestResourceLifecycleManager;
 
-import org.apache.camel.quarkus.component.infinispan.common.InfinispanCommonServerTestResource;
+import java.util.HashMap;
+import java.util.Map;
 
-public class InfinispanServerTestResource extends InfinispanCommonServerTestResource {
+public class InfinispanServerTestResource implements QuarkusTestResourceLifecycleManager {
 
     @Override
     public Map<String, String> start() {
-        Map<String, String> config = super.start();
+        Map<String, String> config = new HashMap<>();
 
         config.put("camel.component.infinispan.autowired-enabled", "false");
-        config.put("camel.component.infinispan.hosts", getServerList());
-        config.put("camel.component.infinispan.username", USER);
-        config.put("camel.component.infinispan.password", PASS);
+        config.put("camel.component.infinispan.hosts", "localhost:31222");
```

Comment:
We need to avoid hard coding `localhost`. It wont work if the tests run against a remote Docker host. You can try getting the actual host value from `DockerClientFactory.instance().dockerHostIpAddress()`.